cypress/integration/buyers-side-pagination.js

```diff
@@ -0,0 +1,184 @@
+const { login } = require('../../helpers/login-helper');
+
+context('Checking Brands and Bidders pagination', () => {
+  describe('Pagination UI', () => {
+    let userSessionToken;
+
+    before(() => {
+      userSessionToken = login('kargoqa@gmail.com', 'K@rgo123!');
+    });
+
+    // This beforeEach is needed because when a subsequent tests hits
+    // the same URL, cookies is not properly retained
+    beforeEach(() => {
+      cy.setCookie('kauth_access', userSessionToken);
+    });
+
+    const rowsSelector = 'data-row';
+    let totalNumberOfEntries = 0;
+
+    it('Navigate to Brands page', () => {
+      cy.visit('');
+      cy.get('[data-qa="nav-bar--buyers"]').click({ force: true }).wait(2500).then(() => {
+        cy.url().should('include', 'buyers-dashboard/brands');
+      });
+      cy.get('.QA-totalPages').then((totalPages) => {
+        totalNumberOfEntries = totalPages.text().split('entries')[0].split('of')[1].replace(' ', '');
+      });
+    });
+
+    it('Verify < arrow is disabled by default', () => {
```

Comment:
same as deal-side review, be more detailed

like:
- each test should be hyphen with the entity.
- i.e. 'Deals: Verify "<" arrow is disabled'